index.js

```diff
@@ -3741,6 +3741,23 @@
   //. [type identifier][] is `'Object'` and the values of its enumerable own
   //. properties are all members of type `a`.
 
+  //# singleton :: String -> a -> StrMap a
+  //.
+  //. Embeds a key and a value into a string map
+  //.
+  //. ```javascript
+  //.
+  //. > S.singleton("foo", 42)
+  //. { foo: 42 }
+  //. ```
+  function singleton(k, v) {
+    var o = {};
+    o[k] = v;
+    return o;
+  }
```

Comment:
For consistency with existing code let's use `key`, `val`, and `strMap` in place of `k`, `v`, and `o`.